plugin/src/test/java/org/opensearch/ml/utils/error/ErrorMessageTests.java

```diff
@@ -28,6 +33,22 @@ public void fetchDetails() {
         assertEquals(errorMessage.fetchDetails(), "illegal state");
     }
 
+    @Test
+    public void fetchDetailsWithPrivateIP() throws UnknownHostException {
```

Comment:
may be add another test, showing without ip address it's not redacting anything else?

Reply:
I think it's already covered by the existing test case fetchDetails() for this case.